### PR TITLE
Create NewFile.md

### PR DESCRIPTION
**Pull request by mattyohe**
Cool!

### CODE REVIEW

**Comment by mattyohe** [comment]
Oh nice! The body of a PR is now just an in-line comment and not some weird other thing.

File: NewFile.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# This is a cool new file
+# This is a really cool new file
```
Comment (mattyohe) [inline NewFile.md]: I'm liking what we're doing here.

Comment (davesroka) [inline NewFile.md]: Pretty good, I think it's ready to go!

Comment (mattyohe) [inline NewFile.md]: On another note: Did you review this copy change with anyone?

Comment (davesroka) [inline NewFile.md]: Oh, you're right.  We need to run this by marketing.   I'm going to revert this change until we get approval.

Comment (davesroka) [inline NewFile.md]: I created a new issue here #5 .  Let's run this by marketing, and then we'll resubmit for pr!
